holmes/main.py

```diff
@@ -456,65 +387,64 @@ def ask(
         False,  # type: ignore
     )
 
-    # Interactive loop
-    if interactive:
-        run_interactive_loop(
-            ai,
-            console,
-            destination,  # type: ignore
-            messages,
-            post_processing_prompt,
-            show_tool_output,
-            interactive_limit,
-        )
-
 
 def run_interactive_loop(
     ai,
     console: Console,
-    destination: DestinationType,
-    messages: list[dict],
+    system_prompt_rendered: str,
+    initial_user_prompt: str,
     post_processing_prompt: Optional[str],
     show_tool_output: bool,
-    limit: int,
 ) -> None:
-    console.print(
-        Rule(
-            "[bold cyan]Entering interactive mode. Type 'exit' or 'quit' to end.[/bold cyan]"
-        )
-    )
-    for _ in range(limit):
-        try:
-            follow_up_prompt = typer.prompt("Follow-up question")
-            if follow_up_prompt.lower() in ["exit", "quit"]:
-                break
-
-            messages.append({"role": "user", "content": follow_up_prompt})
+    initial_messages = [
```

Comment:
A general interactive mode is the user enable the interactive mode with no question?

```
holmes aks --interative
user:
```